### PR TITLE
re-enable python3 tests

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: python
 python:
  - "2.7"
-# - "3.3"
+ - "3.3"
```
Comment (emord) [inline .travis.yml]: Should probably either change this to 3.6, or add more languages (fwiw django 2.0 will only support 3.5+ so 3.5 & 3.6 seem like reasonable choices)

Comment (snopoke) [inline .travis.yml]: updated

File: commcare_export/misc.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def unwrap_val(val):
         if len(val) == 1:
             val = val[0]
         else:
-            val = map(unwrap_val, val)
+            val = list(map(unwrap_val, val))
```
Comment (millerdev) [inline commcare_export/misc.py]: Nitpick: would be better as a list comprehension `[unwrap_val(v) for v in val]` as to only iterate over the items once (in Python 2)

Comment (snopoke) [inline commcare_export/misc.py]: 592d667

**Pull request merged**